source/bin.ts

```diff
@@ -1,16 +1,20 @@
 #!/usr/bin/env node
 
-import validate from './index.js'
+import validate, { IsValidDirectory } from './index.js'
 const path = process.argv[2] || process.cwd()
 
 validate(path)
-	.then((path) => {
-		console.log(`${path} is valid`)
+	.then((valid: IsValidDirectory) => {
```

Comment:
`.then(([isValid, invalidPaths) => {`

that way you don't have to do the arbitrary `valid[0]` and `valid[1]` in the next line, but can do `isValid` instead and `invalidPaths`